test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -575,6 +575,46 @@ TEST(LiteInterpreterTest, TwoSubmodulesModuleInfo) {
   AT_ASSERT(module_debug_info_set == expected_result);
 }
 
+TEST(LiteInterpreterTest, LoadAndRunByteCodeModel) {
+  // Load check in model: sequence.ptl
```

Comment:
Consider adding comments on this test: to test the current code can handle both v4 and v5 models fo backward compatibility.